jnigen/test/kotlin_test/kotlin/src/main/kotlin/com/github/dart_lang/jnigen/SuspendFun.kt

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+ * for details. All rights reserved. Use of this source code is governed by a
+ * BSD-style license that can be found in the LICENSE file.
```

Comment:
Should we also add an example in `jnigen/example`?

Reply:
Good idea, we could add something similar to https://github.com/leighajarett/pedometer there.